cards/cards.yml

```diff
@@ -17,3 +17,21 @@
   links:
     website: "https://golang.org/"
     quickstart: "https://golang.org/doc/install"
+
+"angular ng":
+  name: "Angular"
+  description: "Angular is a TypeScript-based open-source web application framework led by the Angular Team at Google and by a community of individuals and corporations."
```

Comment:
For brevity and to save space, the description should start with `TypeScript-based open-source...` rather than `Angular is a ...` (see previous cards for reference).